plugins/filter/accumulate.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Max Gautier <mg@max.gautier.name>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION = '''
+  name: accumulate
+  short_description: Produce a list of accumulated sums of the input list contents
+  version_added: 10.1.0
+  author: Max Gautier (@VannTen)
+  description:
+    - Passthrough to the L(Python itertools.accumulate function,https://docs.python.org/3/library/itertools.html#itertools.accumulate).
+    - Transforms an input list into the cumulative list of results from applying addition to the elements of the input list.
+    - Addition means the default Python implementation of C('+') for input list elements type.
```

Comment:
The Python string `'+'` is not an operator. Either put `+` in quotes:
```suggestion
    - Addition means the default Python implementation of 'C(+)' for input list elements type.
```
or leave the quotes away for regular code formatting:
```suggestion
    - Addition means the default Python implementation of C(+) for input list elements type.
```

Reply:
For consistency with docs in other plugins, I would recommend option #2.